### ISSUE TITLE
"Something went wrong, please try again"

### ISSUE THREAD

**Issue by peterkisner** [Toybox-Labs/support-page#7]
All the models in the default Toybox library are printing fine. But I have not had any luck importing models into the Creator Space and printing them from there.  I can upload them, but every time I try to print something it gets 50-90% of the way through preparing the file, then sends the message:

"Something went wrong, please try again"

I've tried importing models into Meshmixer, making them solid, doing repairs on them, and reducing the polygon count.  Then exporting them to both STL and OBJ file formats before uploading.  I've tried including support material or excluding support material.   None of these combinations seem to work for Toybox printing.  Always the same message.

Any suggestions?  Is there a maximum file size or anything?

**Comment by nessker** [comment]
I'm really new to the system, but in working with the creators we discovered a glitch in the system. I wasn't able to access a lot of the features and my account was having a lot of issues including uploading to the creator space and working with any files with in it. Today I tried again and all seems to be well. Maybe try again and see if whatever they identified as the problem for me was a pretty global issue.
